DocSum/docker_image_build/build.yaml

```diff
@@ -11,17 +11,27 @@ services:
       context: ../
       dockerfile: ./Dockerfile
     image: ${REGISTRY:-opea}/docsum:${TAG:-latest}
-
-  docsum-ui:
+  docsum-gradio-ui:
     build:
       args:
         http_proxy: ${http_proxy}
         https_proxy: ${https_proxy}
       context: ../ui/
       dockerfile: ./docker/Dockerfile
     extends: docsum
+    image: ${REGISTRY:-opea}/docsum-gradio-ui:${TAG:-latest}
```

Comment:
Use the correct dockerfile name, 
![image](https://github.com/user-attachments/assets/c74727d0-e433-4239-9dfb-1d6cbfc20eee)